docs: clarify cases where streaming don't work (#10451)
sveltejs/kit: documentation/docs/20-core-concepts/20-load.md

</p>
```

> > On platforms that do not support streaming, such as AWS Lambda, responses will be buffered. This means the page will only render once all promises resolve.
> 
> > Make shure that your proxy (e.g. NGINX) do not buffers proxied server response on streaming route, or page will render only after all promises resolve.

Reviewer: ```suggestion
> On platforms that do not support streaming, such as AWS Lambda, responses will be buffered. This means the page will only render once all promises resolve. If you are using a proxy (e.g. NGINX), make sure it does not buffer responses from the proxied server.
```
